src/com/axis/rtspclient/FLVMux.as

```diff
@@ -106,6 +106,16 @@ package com.axis.rtspclient {
       return 1 + 2 + contents.length;
     }
 
+    private function readSE(sps:BitArray):uint {
```

Comment:
I'm guessing this should be understood as 'readSignedExpGolomb' ?
Maybe this is better put in the implementation of BitArray.